### DISCUSSION TITLE
Bootable ISOs from a folder

### DISCUSSION THREAD

**Discussion by classbproject** [aonez/Keka#1246] [General]
I used Keka to extract the Debian 11.6 distribution ISO to a folder since I wanted to edit a few files to enable serial console installation. Would the bootable state be retained if I re-archived this to an ISO using Keka?

**Comment by aonez** [comment]
It will not. You should recreate the original scheme of that ISO image.

Keka creates plain UDF or [ISO9660](https://github.com/aonez/Keka/wiki/Hidden-configuration#compression-and-extraction) images.

**Comment by aonez** [comment]
As an alternative you can recreate it using `xorriso` like this:

```sh
xorriso -as mkisofs \
    -V 'debian-11.6' \
    -e boot/grub/efi.img \
    -no-emul-boot \
    -o <YOUR_DESTIONATION.iso> \
    <YOUR_DEBIAN_FOLDER>
```

You'll need to install `xorriso`. Using [brew](https://brew.sh): `brew install xorriso`

**Reply by classbproject** [reply]
Thank you! This worked 👍🏼
